Makefile

```diff
@@ -4,6 +4,7 @@
 # Source Lists
 SRC = \
 	src/SDL2.cs \
+    src/SDL2_gfx.cs \
```

Comment:
One last thing!

```suggestion
	src/SDL2_gfx.cs \
```

Reply:
Is this indented instead of using spaces? i see, i thought indentation could mess with Makefiles :p

Reply:
Yep, SDL2# uses tabs for everything except sln/csproj files, since that's what Visual Studio sort of forces us into...